lib/window-settings.js

```diff
@@ -1,13 +1,10 @@
 Promise = require('bluebird');
 const electron = require('electron');
-const remote = require('electron').remote;
+const {remote} = require('electron');
 const $ = require('jquery');
-const fs = require('fs');
-const dust = Promise.promisifyAll(require('dustjs-linkedin'));
 const path = require('path');
 const libPath = path.join(__dirname, '..', 'lib');
-
-const selectFolders = electron.remote.require(libPath+'/config.js').selectFolders;
+const config = electron.remote.require(libPath+'/config.js');
```

Comment:
This is somewhat correct. The window-settings should fetch the filePath from the Main instead of knowing it itself. But as I said in a comment above, the config.js module does not need to exist.